gluten-celeborn/src/main/java/org/apache/spark/shuffle/gluten/celeborn/CelebornShuffleManager.java

```diff
@@ -68,46 +67,35 @@ public class CelebornShuffleManager
   private static final CelebornColumnarBatchSerializerFactory columnarBatchSerializerFactory;
 
   static {
+    final String backendName = BackendsApiManager.getBackendName();
```

Comment:
Nit:
It's unnecessary to include backendName in error message. It's already clear to users.